test/modules/core/viewports/viewport.spec.js

```diff
@@ -99,3 +100,16 @@ test('Viewport.containsPixel', t => {
 
   t.end();
 });
+
+test('Viewport.getFrustumPlanes', t => {
+  const viewport = new Viewport(TEST_VIEWPORTS[0].mapState);
```

Comment:
Nice!

I do believe this need to be the `WebMercatorViewport` subclass of `Viewport` to correspond with what `MapView` returns. 

It would probably be safest to let the `MapView` create the `Viewport` instance to avoid second guessing these things in the test suite...